src/library/colordelegate.cpp

```diff
@@ -0,0 +1,43 @@
+#include "library/colordelegate.h"
+
+#include <QPainter>
+#include <QStyle>
+#include <QColor>
+#include <QTableView>
+
+#include "library/trackmodel.h"
+
+namespace {
+    const QRgb hiddenTrackColor = 0xFF000000;
```

Comment:
- No alpha channel, i.e. 0x000000
- kHiddenTrackColor
- This constant should be defined globally and reused for the NULL mapping with the database